.github/workflows/deploy_images.yml

```diff
@@ -156,6 +156,12 @@ jobs:
           name: save-agent-distribution
           path: 'save-cloud/save-agent/build/libs/save-agent-*-distribution.jar'
           retention-days: 5
+      - name: Upload save-demo-agent.kexe
+        uses: actions/upload-artifact@v3
+        with:
+          name: save-demo-agent
+          path: 'save-cloud/save-demo-agent/build/bin/linuxX64/releaseExecutable/save-demo-agent.kexe'
```

Comment:
why `kexe`, but not `distribution.jar` as for save-agent? we will add `save-demo-agent` as dependency to `save-demo`

Reply:
Yep, in next PR I will add Gradle task for that (as well as return agent.toml with server port _maybe_)